build.gradle

```diff
@@ -11,344 +11,333 @@
     }
+  }
 }
 
 spotbugs {
-    effort = 'max'
-    // This makes the "html" reports come out in plain text so you can just open the file in IntelliJ
-    // and look at your bugs instead of having to switch to a browser.
-    extraArgs = [ '-emacs']
+  effort = 'max'
+  // This makes the "html" reports come out in plain text so you can just open the file in IntelliJ
+  // and look at your bugs instead of having to switch to a browser.
+  extraArgs = ['-emacs']
 }
 spotbugsMain {
-    reports {
-        html {
-            enabled = true
-            destination = file("$buildDir/reports/spotbugs/main.html")
-            excludeFilter = file("config/spotbugs/excludeFilter.xml")
-        }
+  reports {
+    html {
+      enabled = true
+      destination = file("$buildDir/reports/spotbugs/main.html")
+      excludeFilter = file("config/spotbugs/excludeFilter.xml")
     }
+  }
 }
 spotbugsTest {
-    reports {
-        html {
-            enabled = true
-            destination = file("$buildDir/reports/spotbugs/test.html")
-            excludeFilter = file("config/spotbugs/excludeFilter.xml")
-        }
+  reports {
+    html {
+      enabled = true
+      destination = file("$buildDir/reports/spotbugs/test.html")
+      excludeFilter = file("config/spotbugs/excludeFilter.xml")
     }
+  }
 }
 
 tasks.withType(JacocoReport) {
-    reports {
-        xml.enabled false
-        csv.enabled false
-        html.destination file("${buildDir}/jacocoHtml")
-    }
+  reports {
+    xml.enabled false
+    csv.enabled false
+    html.destination file("${buildDir}/jacocoHtml")
+  }
 }
 
 jacocoTestCoverageVerification {
-    violationRules {
-        rule {
-            limit {
-                counter = 'BRANCH'
-                value = 'COVEREDRATIO'
-                minimum = 0.2
-            }
-        }
+  violationRules {
+    rule {
+      limit {
+        counter = 'BRANCH'
+        value = 'COVEREDRATIO'
+        minimum = 0.2
+      }
     }
+  }
 }
 check.dependsOn jacocoTestCoverageVerification
 
 repositories {
-    mavenCentral()
-    maven {
-        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/'
-    }
-    maven {
-        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/'
-    }
+  mavenCentral()
+  maven {
+    url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/'
+  }
+  maven {
+    url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/'
+  }
 }
 
 configurations {
-    generatedCompile
-    runtimeClasspath
-}
-
-// Exclude the Spring logger, so everything will use SLF4J
-configurations.all {
```

Comment:
This was required to get the right logging out of Spring. Why can it be removed?